modules/serving-transport-encryption-configuring-selfsigned-clusterissuer.adoc

```diff
@@ -0,0 +1,70 @@
+:_content-type: PROCEDURE
+[id="serving-transport-encryption-configuring-selfsigned-clusterissuer_{context}"]
+= Configuring a SelfSigned ClusterIssuer
+
+[IMPORTANT]
+====
+For simplicity, this procedure uses a `SelfSigned` issuer as root certificate. However, be aware of the implications and limitations of this method documented in the "SelfSigned" section of the `cert-manager` documentation.
+
+If you run your company-specific Private Key Infrastructure (PKI), Red Hat recommends the CA issuer. Refer to the cert-manager documentation on CA issuers for more details. However, you can use any other issuer that allows signing of certificates as described in the "Choice of a certificate issuer" section.
+====
+
+.Procedure
+
+. Create a `SelfSigned` `ClusterIssuer`:
++
+[source,yaml]
+----
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: knative-serving-selfsigned-issuer
+spec:
+  selfSigned: {}
+----
+
+. Apply the `ClusterIssuer` resource:
++
+[source,yaml]
+----
+$ oc apply -f <filename>
+----
+
+. Create a root certificate using the previously created `SelfSigned ClusterIssuer`:
++
+[source,yaml]
+----
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: knative-serving-selfsigned-ca
+  namespace: cert-manager <1>
+spec:
+  secretName: knative-serving-ca <2>
+
+  isCA: true
+  commonName: selfsigned-ca
+  privateKey:
+    algorithm: ECDSA
+    size: 256
+
+  issuerRef:
+    name: knative-serving-selfsigned-issuer
+    kind: ClusterIssuer
+    group: cert-manager.io
+----
+<1> The {cert-manager-operator} namespace, `cert-manager` by default.
+<2> Secret name later used for the `ClusterIssuer` for Serving.
+
+. Apply the `Certificate` resource:
++
+[source,yaml]
+----
+$ oc apply -f <filename>
+----
+
+[id="additional-resources_serving-transport-encryption-configuring-selfsigned-clusterissuer"]
+[role="_additional-resources"]
+== Additional resources
+* link:https://cert-manager.io/docs/configuration/selfsigned/[cert-manager documentation on self-signed issuers]
+* link:https://cert-manager.io/docs/configuration/ca/[cert-manager documentation on CA issuers]
```

Comment:
Rather than including these as additional resources, I would include them as links directly in the text as shown in the comments / example I provided for selfsigned docs. I think having these links directly where they are mentioned since we're able to will be more helpful to users.